index.bs

```diff
@@ -455,6 +471,36 @@ dictionary XRSessionCreationOptions {
 };
 </pre>
 
+XRRenderState {#xrrenderstate-interface}
+-------------
+
+There are multiple values that developers can configure which affect how the session's output is composited. These values are tracked by an {{XRRenderState}} object.
+
+<pre class="idl">
+dictionary XRRenderStateInit {
+  double depthNear;
+  double depthFar;
+  XRLayer baseLayer;
+};
+
+[SecureContext, Exposed=Window] interface XRRenderState {
+  readonly attribute double depthNear;
+  readonly attribute double depthFar;
+  readonly attribute XRLayer? baseLayer;
+};
+</pre>
+
+<div class="algorithm" data-algorithm="initialize-renderstate">
+
+When an {{XRRenderState}} object is created, the user agent MUST <dfn>initialize the render state</dfn> by running the following steps:
```

Comment:
I don't think we use this algorithm?

Also, this is better achieved by setting these defaults in the dictionary's webidl directly imo

Reply:
The reason this isn't set on the dictionary is that the dictionary elements are optional.  When `updateRenderState()` is invoked, only the values that are present are applied.

Reply:
Ah, I misinterpreted this, my bad.

We should explicitly mention that this is invoked during XRSession construction